package.json

```diff
@@ -427,6 +427,16 @@
 					"type": "boolean",
 					"default": false,
 					"description": "%asciidoc.useWorkspaceRoot.desc%"
+				},
+				"asciidoc.antora": {
+					"type": "boolean",
+					"default": false,
+					"description": "%asciidoc.antora.desc%"
+				},
+				"asciidoc.antoraConfigIsSet": {
```

Comment:
As far as I know, we don't need to explicitly declare this configuration since we don't want to configure it using "User Settings".